packages/venia-ui/lib/components/FilterModal/CurrentFilters/currentFilter.js

```diff
@@ -8,8 +8,7 @@ import Trigger from '../../Trigger';
 import defaultClasses from './currentFilter.css';
 
 const CurrentFilter = props => {
-    const { group, groupName, item, removeItem } = props;
```

Comment:
You can remove `groupName` from the parent, `CurrentFilters`. And if you do that, I believe you can get rid of `filterNames` being passed in from `FilterModal`.